Owncloud iOs Client/Login/Login/UniversalLoginViewController.swift

```diff
@@ -658,23 +656,16 @@ public enum TextfieldType: String {
         
         self.setNetworkActivityIndicator(status: false)
         
-        let userCredDto :OCCredentialsDto =  OCCredentialsDto()
-        userCredDto.accessToken = cookieString;
-        userCredDto.authenticationMethod = .SAML_WEB_SSO;
-        
-        if self.loginMode == .expire {
-            let app: AppDelegate = (UIApplication.shared.delegate as! AppDelegate)
-            userCredDto.userName = app.activeUser.username
-        }
-        
+        self.userNewCredentials =  OCCredentialsDto()
+        self.userNewCredentials.accessToken = cookieString;
+        self.userNewCredentials.authenticationMethod = .SAML_WEB_SSO;
         
         if cookieString == nil || cookieString == "" {
```

Comment:
we could extract this check into a String extension. Doing this we can use this code in other many places.